src/psyclone/psyir/nodes/reference.py

```diff
@@ -120,33 +120,6 @@ def reference_accesses(self, var_accesses):
         '''
         var_accesses.add_access(self.name, AccessType.READ, self)
 
```

Comment:
Could you please explain how removing check_declared is related to this PR. Sorry if I'm slow on the uptake.

Reply:
Strictly speaking it's not - I just realised that it is dead code while updating the tests.